lumigator/backend/backend/api/http_headers.py

```diff
@@ -1,3 +1,4 @@
 class HttpHeaders:
     """HttpHeaders represents constants used in HTTP headers"""
+
```

Comment:
same here re: ruff

Reply:
Yes, this stuff was either never edited after we added ruff, or was committed using the `--no-verify` tag.